Update delimit-volume-allocation.md (#3688)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/storage/storage-spaces/delimit-volume-allocation.md

### New: delimited allocation

With delimited allocation, you specify a subset of servers to use (minimum four). The volume is divided into slabs that are copied three times, like before, but instead of allocating across every server, **the slabs are allocated only to the subset of servers you specify**.

> For example, if you have an 8 node cluster (nodes 1 through 8) you can specify a volume to be located on only on disks in nodes 1,2,3,4.

Reviewer: ```suggestion
For example, if you have an 8 node cluster (nodes 1 through 8) you can specify a volume to be located on disks only in nodes 1, 2, 3, 4.
```

> For example, if you have an 8 node cluster (nodes 1 through 8) you can specify a volume to be located on only on disks in nodes 1,2,3,4.

Reviewer: This line still needs normal spacing between the numerals:
```suggestion
For example, if you have an 8 node cluster (nodes 1 through 8) you can specify a volume to be located on only on disks in nodes 1, 2, 3, 4.
```
@JohanFreelancer9 : Do you have time to suggest any other grammatical or readability improvements in this sentence?

Reviewer 2: @illfated  Looks good to me.

> For example, if you have an 8 node cluster (nodes 1 through 8) you can specify a volume to be located on only on disks in nodes 1,2,3,4.

Reviewer 2: ```suggestion
For example, if you have an eight-node cluster (nodes 1 through 8) you can specify a volume to be located on only on disks in nodes 1,2,3,4.
```

#### Advantages

> With the example allocation, the volume is likely to survive three concurrent failures. If nodes 1,2, and 6 go down, only 2 of the nodes that hold the 3 copies of data for the volume are down and the volume will stay online.

Reviewer: ```suggestion
With the example allocation, the volume is likely to survive three concurrent failures. If nodes 1, 2, and 6 go down, only 2 of the nodes that hold the 3 copies of data for the volume are down and the volume will stay online.
```
(added space)

> With the example allocation, the volume is likely to survive three concurrent failures. If nodes 1,2, and 6 go down, only 2 of the nodes that hold the 3 copies of data for the volume are down and the volume will stay online.

Reviewer 2: ```suggestion
With the example allocation, the volume is likely to survive three concurrent failures. If nodes 1, 2, and 6 go down, only 2 of the nodes that hold the 3 copies of data for the volume are down and the volume will stay online.
```

   > [!TIP]
   > In Storage Spaces Direct, the term 'Storage Scale Unit' refers to all the raw storage attached to one server, including direct-attached drives and direct-attached external enclosures with drives. In this context, it's the same as 'server'.

> 2. Specify which servers to use with the new `-StorageFaultDomainsToUse` parameter and by indexing into `$Servers`. For example, to delimit the allocation to the first, second, third, and fourt servers (indices 0, 1, 2, and 3):

Reviewer: typo
```suggestion
2. Specify which servers to use with the new `-StorageFaultDomainsToUse` parameter and by indexing into `$Servers`. For example, to delimit the allocation to the first, second, third, and fourth servers (indices 0, 1, 2, and 3):
```

> 2. Specify which servers to use with the new `-StorageFaultDomainsToUse` parameter and by indexing into `$Servers`. For example, to delimit the allocation to the first, second, third, and fourt servers (indices 0, 1, 2, and 3):

Reviewer 2: ```suggestion
2. Specify which servers to use with the new `-StorageFaultDomainsToUse` parameter and by indexing into `$Servers`. For example, to delimit the allocation to the first, second, third, and fourth servers (indices 0, 1, 2, and 3):
```

VirtualDiskFriendlyName TotalFootprint Server1 Server2 Server3 Server4 Server5 Server6
----------------------- -------------- ------- ------- ------- ------- ------- -------
MyVolume                300 GB         100 GB  100 GB  100 GB  100 GB  0       0      
```

> Note that only Server1, Server2, Server 3, and Server4 contains slabs of *MyVolume*.

Reviewer: ```suggestion
Note that only Server1, Server2, Server3, and Server4 contain slabs of *MyVolume*.
```

### Change a delimited allocation

Use the new `Add-StorageFaultDomain` and `Remove-StorageFaultDomain` cmdlets to change how the allocation is delimited.

For example, to move *MyVolume* over by one server:

> 1. Specify that the Fifth server **can** store slabs of *MyVolume*:

Reviewer: ```suggestion
1. Specify that the fifth server **can** store slabs of *MyVolume*:
```
(lowercase)

VirtualDiskFriendlyName TotalFootprint Server1 Server2 Server3 Server4 Server5 Server6
----------------------- -------------- ------- ------- ------- ------- ------- -------
MyVolume                300 GB         0       100 GB  100 GB  100 GB  100 GB  0      
```

> Note that Server1 does not contain slabs of *MyVolume* anymore – instead, Server05 does.

Reviewer: (use same name as in the table)
```suggestion
Note that Server1 does not contain slabs of *MyVolume* anymore – instead, Server5 does.
```

## Best practices

Here are the best practices to follow when using delimited volume allocation:

### Choose four servers

> Delimit each three-way mirror volume to foure servers, not more.

Reviewer: (typo)
```suggestion
Delimit each three-way mirror volume to four servers, not more.
```

### Balance storage

Balance how much storage is allocated to each server, accounting for volume size.

> ### Stager delimited allocation volumes

Reviewer 2: ```suggestion
### Stagger delimited allocation volumes
```

To maximize fault tolerance, make each volume's allocation unique, meaning it does not share *all* its servers with another volume (some overlap is okay). 

> For example on an 8 node system:

Reviewer 2: ```suggestion
For example on an eight-node system:
```

> Volume 1: Servers 1,2,3,4

Reviewer 2: ```suggestion
Volume 1: Servers 1, 2, 3, 4
```

> Volume 2: Servers 5,6,7,8

Reviewer 2: ```suggestion
Volume 2: Servers 5, 6, 7, 8
```

> Volume 3: Servers 3,4,5,6

Reviewer 2: ```suggestion
Volume 3: Servers 3, 4, 5, 6
```

> Volume 4: Servers 1,2,7,8

Reviewer 2: ```suggestion
Volume 4: Servers 1, 2, 7, 8
```

### Up to two failures is always okay

Every three-way mirror volume can survive up to two failures at the same time, regardless of its allocation. If two drives fail, or two servers fail, or one of each, every three-way mirror volume stays online and accessible, even with regular allocation.

### More than half the cluster failing is never okay

Conversely, in the extreme case that more than half of servers or drives in the cluster fail at once, [quorum is lost](understand-quorum.md) and every three-way mirror volume goes offline and becomes inaccessible, regardless of its allocation.

### What about in between?

> If three or more failures occur at once but at least half of servers and drives are still up, volumes with delimited allocation may stay online and accessible, depending on which servers have failures. 

Reviewer: ```suggestion
If three or more failures occur at once, but at least half of the servers and the drives are still up, volumes with delimited allocation may stay online and accessible, depending on which servers have failures.
```
